server/StudySharp.ApplicationServices/Commands/UpdatePracticalBlockCommand.cs

```diff
@@ -28,18 +44,7 @@ public UpdatePracticalBlockCommandHandler(StudySharpDbContext sharpDbContext)
 
         public async Task<OperationResult> Handle(UpdatePracticalBlockCommand request, CancellationToken cancellationToken)
         {
-            var courseExistent = await _context.Courses.AnyAsync(_ => _.Id == request.CourseId, cancellationToken);
-            if (!courseExistent)
-            {
-                return OperationResult.Fail(string.Format(ErrorConstants.EntityNotFound, nameof(Course), nameof(Course.Id), request.CourseId));
-            }
-
-            var practicalBlock = await _context.PracticalBlocks.FirstOrDefaultAsync(_ => _.Id == request.Id, cancellationToken);
-            if (practicalBlock == null)
-            {
-                return OperationResult.Fail(string.Format(ErrorConstants.EntityNotFound, nameof(PracticalBlock), nameof(PracticalBlock.Id), request.Id));
-            }
-
+            var practicalBlock = await _context.PracticalBlocks.FindAsync(request.Id, cancellationToken);
             practicalBlock.Name = request.Name;
```

Comment:
Should also add validation for Name (NotEmpty, max length 120) and Description (max length 512)